src/index.js

```diff
@@ -204,11 +176,20 @@ module.exports = (
       minimize: false,
       moduleIds: 'deterministic',
       chunkIds: 'deterministic',
-      mangleExports: false
+      mangleExports: false,
+      concatenateModules: false,
+      innerGraph: false,
+      sideEffects: false
```

Comment:
😢

Reply:
Correctness is more important than bundle size.

Reply:
I actually went ahead and reenabled these options and all the tests seem to pass. I suppose I am familiar with Rollup treeshaking having correctness issues, apologies for the webpack projection :)